openmc/element.py

```diff
@@ -1,9 +1,9 @@
 from collections import OrderedDict
-import os
 import re
 from xml.etree import ElementTree as ET
 
 import openmc.checkvalue as cv
+import openmc
```

Comment:
Is it worth importing just the config part of openmc
```suggestion
from openmc.config as config
```

Reply:
Same comment re: confusion over local vs global variables